runtime/manual/basics/modules/private_registries.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Private Registries"
```

Comment:
To make it clearer that we're talking about npm, can we rename this page to something like `npm private registries`? Also, does it make sense to be in the "basic" page and not in the "Node compat" section?